src/quo/components/list_items/token_network/view.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo.components.list-items.token-network.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.list-items.preview-list.view :as preview-list]
+    [quo.components.list-items.token-network.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.common :as common]
+    [quo.foundations.resources :as resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn- info
+  [{:keys [token networks]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (resources/get-token token)
```

Comment:
hmm I would remove the call to resources and instead just pass `token` or `token-source`. this way we can handle that on the application code side

Reply:
maybe it could also do like
`(or (:source token) token)` so that it works with both structures which are common in the codebase.
e.g  the values could be `{:source "..."}` or `"..."`